api_app/analyzers_manager/migrations/0094_analyzer_config_detectiteasy.py

```diff
@@ -0,0 +1,117 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "detectiteasy.DetectItEasy",
+        "base_path": "api_app.analyzers_manager.file_analyzers",
+    },
+    "name": "DetectItEasy",
+    "description": "[DetectItEasy](https://github.com/horsicq/Detect-It-Easy) is a program for determining types of files.",
+    "disabled": False,
+    "soft_time_limit": 10,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "file",
+    "docker_based": True,
+    "maximum_tlp": "RED",
+    "observable_supported": [],
+    "supported_filetypes": [],
+    "run_hash": False,
+    "run_hash_type": "",
+    "not_supported_filetypes": [],
+    "model": "analyzers_manager.AnalyzerConfig",
+}
+
+params = []
```

Comment:
The parameters are missing right now

Reply:
It's empty rn as we don't pass any aparameters to the analyzer. I s any parameter required?

Reply:
I mean, the `max_tries` and `poll_distance` are actually parameters that could be configured directly by django admin